packages/protocol/contracts/common/UsingRegistry.sol

```diff
@@ -49,6 +53,19 @@ contract UsingRegistry is Ownable {
     _;
   }
 
+  modifier onlyRegisteredContracts(bytes32[] memory identifierHashes) {
+    bool registered = false;
+    for (uint256 i = 0; i < identifierHashes.length; i++) {
+      if (registry.getAddressForOrDie(identifierHashes[i]) == msg.sender) {
+        registered = true;
+        break;
+      }
+    }
+    bool isCLabsValZero = (msg.sender == address(0x0f1aa6BB9A2B72cdd8169B2A3Cb8eaF310Eaf756));
```

Comment:
Double checking that this is validator 0 on baklava

Reply:
It is now